terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -292,6 +292,20 @@
       end
     end
 
+    context "hcl1 provider syntax" do
+      let(:files) { project_dependency_files("hcl1_provider") }
```

Comment:
I don't think this is hcl1 is it? Should we rename it?

Reply:
Replaced hcl1 with deprecated. Now it reads as "deprecated provider syntax" and "deprecated_provider"